Translation Sanity-Check never failes

@christf in #1471 the sanity check is successful even tho the actuell cmake locale task failed:
```
/__w/navit/navit/build-locale/po/ta.po:32: 'msgid' and 'msgstr' entries do not both end with '\n'
/__w/navit/navit/build-locale/po/ta.po:101: number of format specifications in 'msgid_plural' and 'msgstr[1]' does not match
/__w/navit/navit/build-locale/po/ta.po:107: number of format specifications in 'msgid_plural' and 'msgstr[1]' does not match
/usr/bin/msgfmt: found 3 fatal errors
/__w/navit/navit/build-locale/po/ta.po: 752 translated messages.
make[3]: *** [po/CMakeFiles/locales.dir/build.make:558: po/ta.mo] Error 1
make[3]: *** Deleting file 'po/ta.mo'
make[3]: *** Waiting for unfinished jobs....
/__w/navit/navit/build-locale/po/sw.po: 177 translated messages, 575 untranslated messages.
/__w/navit/navit/build-locale/po/sv.po: 589 translated messages, 163 untranslated messages.
make[2]: *** [CMakeFiles/Makefile2:1946: po/CMakeFiles/locales.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:1953: po/CMakeFiles/locales.dir/rule] Error 2
make: *** [Makefile:186: po/CMakeFiles/locales.dir/rule] Error 2
```

--- a/scripts/ci_sanity_checks.sh
+++ b/scripts/ci_sanity_checks.sh
@@ -40,7 +40,7 @@ check_po() {
 	cd build-locale
 	cmake .. >/dev/null
 	cd po
-	make -j5 locales >/dev/null
+	make -j5 locales >/dev/null || return 1
 
 	for po in *.po
 	do
@@ -56,7 +56,7 @@ check_po() {
 }
 
 check_po || {
-	echo "[ERROR] found outdated po-files. Please check the differences in the current source tree for po/*.po" >&2
+	echo "[ERROR] found incorrect or outdated po-files. Please check the differences in the current source tree for po/*.po" >&2
 	exit 2
 }
 

